packages/libsql-client/examples/sync_vector.js

```diff
@@ -0,0 +1,51 @@
+import { createClient } from "@libsql/client";
+import reader from "readline-sync";
```

Comment:
This example suggests a [deprecated lib](https://github.com/anseki/readline-sync).